docs/Framework.md

```diff
@@ -100,7 +100,11 @@ where **elementId** is a delimited string of the form `nTm`, where
 - `T` is the elementId type key (p, m or e).
 - `m` is the ordinal of the definition within the class
 
-Every property, method or event MUST be uniquely identified in a control class using element ids.
+Every property, method or event MUST be uniquely identified in a control class using element ids. The `level` inside an **elementId** MUST match the inheritance level of the class which defines that element. Inheritance levels can be calculated by taking the [classId](Framework.md#ncclassid) of a class and counting the indexes after removing all authority keys.
```

Comment:
In the framework document we have 'element id' and 'elementId' as well as 'NcElementId'

Perhaps the 'elementId' here should be an 'NcElementId' (or 'element id')?

Reply:
I have changed it to "element id" and added a link to the model definition of NcElementId.